x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/context/waterfall_chart.tsx

```diff
@@ -10,6 +10,17 @@ import { WaterfallData, WaterfallDataEntry, WaterfallMetadata } from '../types';
 import { OnSidebarClick, OnElementClick, OnProjectionClick } from '../components/use_flyout';
 import { SidebarItems } from '../../step_detail/waterfall/types';
 
+export type MarkerItems = Array<{
+  id:
+    | 'domContentLoaded'
```

Comment:
Note:  we also get `cls`, but it's not a relative offset or trace. Its an actual value and not presentable in the waterfall.